packages/network/src/logic/PerStreamMetrics.ts

```diff
@@ -9,9 +9,11 @@ interface AllMetrics<M> {
 }
 
 interface Metric {
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
Hmmm I actually believe this class may not be used anymore

Reply:
Removed PerStreamMetrics. The Node class recorded some metrics, but `report()` was never called.